sdk/flow/confirm.go

```diff
@@ -51,7 +52,8 @@ func (item *ConfirmerItem) Confirm(ctx *cli.Context) (bool, error) {
 	fmt.Scanln(&option)
 	action, err := item.action(option)
 	if err != nil {
-		return false, errors.Trace(err)
+		fmt.Fprintf(os.Stderr, "no action found for option %q", option)
```

Comment:
```suggestion
		util.Logger.Warnf("no action found for option %q", option)
```

From `"github.com/codelingo/lingo/app/util"`

Reply:
Will this notify the user though? Or just if debug flag is set?

Reply:
Ah OK, yeah I missed the point here.